framework/src/modules/http_api/controllers/delegates.js

```diff
@@ -15,18 +15,16 @@
 'use strict';
 
 const _ = require('lodash');
-const Bignum = require('../../helpers/bignum.js');
-const swaggerHelper = require('../../helpers/swagger');
-const BlockReward = require('../../logic/block_reward');
-const apiCodes = require('../../helpers/api_codes.js');
-const ApiError = require('../../helpers/api_error.js');
-const { calculateApproval } = require('../../helpers/http_api');
-const slots = require('../../helpers/slots.js');
+const Bignum = require('../helpers/bignum.js');
+const swaggerHelper = require('../helpers/swagger');
+const apiCodes = require('../helpers/api_codes.js');
+const ApiError = require('../helpers/api_error.js');
+const { calculateApproval } = require('../helpers/http_api');
+const slots = require('../helpers/slots.js');
 // Private Fields
-let modules;
 let storage;
-let blockReward;
 let logger;
+let channel;
 const { EPOCH_TIME, ACTIVE_DELEGATES } = global.constants;
```

Comment:
this will be obtained from config in the future?

Reply:
Yes with changes in other PR you will get `scope.config.constants`